gradle/libs.versions.toml

```diff
@@ -16,42 +16,42 @@ jakarta-inject-api = "2.0.1"
 jakarta-inject-tck = "2.0.1"
 javax-inject = "1"
 javax-persistence = "2.2"
-jetty = "9.4.38.v20210224"
+jetty = "9.4.44.v20210927"
 jmh = "1.29"
 jsr107 = "1.1.1"
 junit4 = "4.13.2"
 javax-el = "3.0.1-b12"
 javax-el-impl = "2.2.1-b05"
-kotest-junit5 = "4.6.0"
+kotest-junit5 = "4.6.4"
 logbook-netty = "2.4.1"
 log4j = "2.17.1"
 mongo-javadriver = "3.12.10"
-mysql-driver = "8.0.22"
+mysql-driver = "8.0.27"
 powermock = "2.0.9"
 rxjava3 = "3.0.13"
 selenium = "3.141.59"
 smallrye = "5.1.0"
-systemlambda = "1.2.0"
-vertx = "3.9.4"
+systemlambda = "1.2.1"
+vertx = "3.9.12"
 wiremock = "2.27.2"
 
 #
 # Versions which start with managed- are managed by Micronaut in the sense
 # that they will appear in the Micronaut BOM as <properties>
 #
 managed-azure-function = "1.4.2"
-managed-brave-instrumentation = "5.13.3"
+managed-brave-instrumentation = "5.13.7"
 managed-brave-opentracing = "1.0.0"
 managed-commons-dbcp = "2.8.0"
 managed-dekorate = "1.0.3"
-managed-elasticsearch = "7.16.1"
+managed-elasticsearch = "7.16.3"
```

Comment:
This is managed by micronaut-elasticsearch https://github.com/micronaut-projects/micronaut-elasticsearch/blob/master/gradle.properties#L19